doc-content/enterprise-only/installation/clustering-quartz-proc.adoc

```diff
@@ -0,0 +1,103 @@
+[id='clustering-quartz-proc_{context}']
+= Configuring Quartz timer service
+
+When you run {KIE_SERVER} in a cluster you must configure the Quartz timer service.
+
+Before you configure a database on your application server, you must prepare the database for Quartz to create Quartz tables, which will hold the timer data, and the Quartz definition file.
+
+.Prerequisites
+
+* A supported non-JTA data source is available. For example, a PostgreSQL data source.
+
+* The database must be connected to your application server.
+
+.Procedure
+
+. Create Quartz tables in your database to allow timer events synchronization using the DDL script for your database.
++
+The DDL script is available in the extracted supplementary ZIP archive in `_QUARTZ_HOME_/docs/dbTables`.
+
+. Create the Quartz configuration file `quartz-definition.properties` in the `_JBOSS_HOME_/_MODE_/configuration/` directory and add the following example content:
++
```

Comment:
+ appears in rendered output. Check.
example content:
So if this is example content why would I want to enter it? Do I enter content similar to the following?

Reply:
Will remove "example" and will add explanations and alternatives for 7.5 Jira